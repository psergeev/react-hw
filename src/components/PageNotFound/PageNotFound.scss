html, body, #app {
    font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
    margin: 0;
    min-width: 600px;
    height: 100%;
}

.page-not-found {
    height: 100%;
    display: flex;
    justify-content: center;
    align-items: center;
    font-size: 20px;

    div {
        text-align: center;

        h1 {
            font-size: 50px;
        }
    }
}
