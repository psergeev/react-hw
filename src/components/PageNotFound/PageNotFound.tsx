import * as React from 'react';
import './PageNotFound.scss';

export default React.memo(() => (
    <div className="page-not-found">
        <div>
            <h1>404 Error</h1>
            Page Not Found
        </div>
    </div>
));
