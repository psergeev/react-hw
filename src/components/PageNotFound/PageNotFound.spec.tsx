import * as React from 'react';
import * as renderer from 'react-test-renderer';
import PageNotFound from '.';

test('PageNotFound component works as expected', () => {
    const component = renderer.create(
        <PageNotFound />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
