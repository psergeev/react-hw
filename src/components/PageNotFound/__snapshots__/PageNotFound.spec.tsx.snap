// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component works as expected 1`] = `
<div
  className="page-not-found"
>
  <div>
    <h1>
      404 Error
    </h1>
    Page Not Found
  </div>
</div>
`;
